### PR TITLE
working txgen for 1 shard

### PR DESCRIPTION
**Pull request by alajko**
## Issue

Current TXGen is not setup to send transactions to testnet. In order to send a volume of txns via txgen we need to sync state to current blockchain. This TXGen adds ability to send transactions directly to tesnet AND send large vollume of txns.

## TODO NEXT

- [ ] Test: Stability of TXGen over hours of sending a large vollume of transactions.
- [ ] Test: Stability of TXGen in sending a larger vollume of txns (>500) per batch.
- [ ] Archival Feature: Ability of TXGen to start and stop without having to sync entire blockchain everytime.
- [ ] Agility Feature: Agility of TXGen to syncto  current state quickly and generate valid txns while syncing to entire blockchain in background.
- [ ] Stability Feature: Stability of TXGen when it has to sync to a blockchain with height in 1000's.

### CODE REVIEW

File: .hmy/wallet.ini [hunk 1]
```diff
@@ -1,14 +1,40 @@
 [default]
-bootnode = /ip4/100.26.90.187/tcp/9876/p2p/QmZJJx6AdaoEkGLrYG4JeLCKeCKDjnFz2wfHNHxAqFSGA9
-bootnode = /ip4/54.213.43.194/tcp/9876/p2p/QmQayinFSgMMw5cSpDUiD9pQ2WeP6WNmGxpZ6ou3mdVFJX
-shards = 1
+bootnode = /ip4/100.26.90.187/tcp/9874/p2p/Qmdfjtk6hPoyrH1zVD9PEH4zfWLo38dP2mDvvKXfh3tnEv
```
Comment (LeoHChen) [inline .hmy/wallet.ini]: don't check-in wallet.ini change.

File: node/node_syncing.go [hunk 2]
```diff
@@ -103,12 +103,16 @@ func (node *Node) SupportSyncing() {
...
 	go node.SendNewBlockToUnsync()
-
 	if node.NodeConfig.Role() != nodeconfig.ShardLeader && node.NodeConfig.Role() != nodeconfig.BeaconLeader {
 		go node.DoSyncing(node.blockchain, node.Worker, node.GetSyncingPeers, true)
 	}
 }
 
+// GetSync gets sync-ed to blockchain without joining consensus
+func (node *Node) GetSync() {
+	go node.DoSyncing(node.blockchain, node.Worker, node.GetBeaconSyncingPeers, false) //Don't join consensus
```
Comment (LeoHChen) [inline node/node_syncing.go]: why redundant go routine call here?  You already called ```go GetSync```.

**Comment by LeoHChen** [comment]
also, the commit has to be squashed.

File: node/node_syncing.go (hunk 2 above)
Comment (alajko) [reply node/node_syncing.go]: sorry got copied over from a old commit.

Comment (alajko) [reply node/node_syncing.go]: Actually I am NOT doing `go GetSync`. I fixed it correctly.

**Comment by alajko** [comment]
Jenkins, test this please.

**Comment by alajko** [comment]
Jenkins, test this please.

**Comment by alajko** [comment]
Jenkins, test this please.

**Pull request closed without merging**